use-cases/inferencing/serving/vllm/hyperdiskML/README.md

```diff
@@ -0,0 +1,309 @@
+# Distributed Inferencing on vLLM using HyperdiskML
+
+This guide demonstrates how to serve a model with vllm using hyperdiskML. By the end of this guide, you should be able to perform the following steps:
+
+- Create a hyperdiskML for the LLM model weights
+- Deploy a vLLM container to your cluster to host your model
+- Use vLLM to serve the fine-tuned Gemma model
+- View Production metrics for your model serving
+- Use custom metrics and Horizontal Pod Autoscaler (HPA) to scale your model
+
+## Prerequisites
```

Comment:
a note here, or somewhere in the README that discusses performance benefits on larger models.